packages/react-components/react-aria/etc/react-aria.api.md

```diff
@@ -6,8 +6,10 @@
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 7 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27753&commitId=c611763e7d45a5e1a0b6b05a2efd107ef6c52096&env=prod)

✅ There was 0 screenshots added, 3 screenshots removed, 1929 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.non-modal - Dark Mode.chromium.png|669524| Changed|
|Dialog.non-modal - High Contrast.chromium.png|665819| Changed|
|Dialog.non-modal - RTL.chromium.png|691260| Changed|
|Dialog.non-modal.chromium.png|691260| Changed|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|0| Removed|
|DatePicker Compat.default - High Contrast.opened.chromium.png|0| Removed|
|DatePicker Compat.default.opened.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>